tests/test_data/config_dirs/second_one_dir/y.conf

```diff
@@ -1,2 +1,2 @@
 [storage]
-path = "/latest_path"
\ No newline at end of file
+path = "latest_path"
```

Comment:
I recommend adding something in here so that we can test that `y.conf` is actually read and processed if `second_one_dir` is provided, even if it comes after `one_dir`, as long as `one_dir` doesn't have it's own `y.conf`.